### ISSUE TITLE
clantool

### ISSUE THREAD

**Issue by Ryach** [mikahanninen/clantool#1]
Hi I am new here, sorry if I write in wrong place.
I install yours program on my server, I think is everything properly set-up, but after hit the update clan I see name of my members, but noting coming to my database, the same happens when I try upload new replay from battle, looks like is updated  but still nothing change in my database.
I will be grateful for any hints
Thank you

**Comment by Snuupy** [comment]
Yeah, I also had that problem, I contacted mikahanninen over twitter but I
was unable to resolve the issue.

**Comment by mikahanninen** [comment]
I will try to check this soon as I can (which might be not soon, this week is busy and next week I'm on vacation).

**Comment by mikahanninen** [comment]
I added small function to output errors related to mysql --- try that and see if you get any information (should be outputted on screen).
